deluge/common.py

```diff
@@ -55,6 +55,25 @@
 
 log = logging.getLogger(__name__)
 
+# A TLS ciphers list.
+# Sources for more information on TLS ciphers:
+# - https://wiki.mozilla.org/Security/Server_Side_TLS
+# - https://www.ssllabs.com/projects/best-practices/index.html
+# - https://hynek.me/articles/hardening-your-web-servers-ssl-ciphers/
+#
+# The general intent is:
+# - prefer cipher suites that offer perfect forward secrecy (ECDHE),
+# - prefer AES-GCM over ChaCha20 because hardware-accelerated AES is common,
+# - disable NULL authentication, MD5 MACs and DSS for security reasons.
+TLS_CIPHERS = ':'.join(['ECDH+AESGCM',
```

Comment:
Should avoid using hanging indents so drop the first item onto a new line and dedent.

Also, include a reference link to where this list originated from for aid future devs looking at this code.

Reply:
Done.